Clinical-Genomics/scout
# Pagination revert

This PR adds a functionality or fixes a bug.

**How to test**:
1. check through all variants pages and make sure all pagination buttons (Next, First, final-First) work, and that the corresponding page layout looks ok.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
